tracker/ffx_rng_tracker/data/equipment_shops.py

```diff
@@ -53,4 +53,4 @@ def _get_equipment_shops(file_path: str) -> dict[str, tuple[Equipment]]:
     return shops
 
 
-EQUIPMENT_SHOPS = _get_equipment_shops("tracker\\data\equipment_shops.csv")
+EQUIPMENT_SHOPS = _get_equipment_shops("tracker\\data\\equipment_shops.csv")
```

Comment:
The real fix to this is to use `os.path.join`. Feel free to do hold off until a follow up PR.

Reply:
This is actually part of the RNG tracker "library" we are using, so i was a bit hesitant to make that change but flake8 kept complaining about it (invalid escape sequence \e or something like that).